storey/table.py

```diff
@@ -1022,7 +1023,8 @@ def __init__(self, max_value=None, set_data=None, set_time=None):
 
     def aggregate(self, time, value):
         if time is not None and not math.isnan(value) and (self.time is None or time < self.time):
-            self._set_value(value)
+            if math.isnan(self.value) and math.isnan(self.default_value):
```

Comment:
why does it matter what's the default value?

Reply:
actually, wouldn't it better to check 
if self.value == self.default_value: ...
instead of assuming nan means it's not set

Reply:
NaN, by IEEE definition: comparing NaN == NaN will always return false:
https://en.wikipedia.org/wiki/NaN  look for "Comparison with NaN"